giraph-examples/src/main/java/org/apache/giraph/examples/feature_diffusion_utils/datastructures/DiffusionVertexValue.java

```diff
@@ -0,0 +1,117 @@
+package org.apache.giraph.examples.feature_diffusion_utils.datastructures;
+
+import org.apache.hadoop.io.Writable;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+// DEBUG
+//import java.util.LinkedList;
+import java.util.LinkedList;
+
+public class DiffusionVertexValue implements Writable {
+
+    protected int vertexThreshold;
+    protected int label;
+    protected double currentActivationProbability=0.2;
```

Comment:
I'm confused. This constant is also initialized in DiffusionMigrationSimulationComputation. Please define constants in a single place